spec/inputs/eventlog_spec.rb

```diff
@@ -1,7 +1,11 @@
 require "logstash/devutils/rspec/spec_helper"
-require 'logstash/inputs/eventlog'
```

Comment:
the test is not run because it is excluded at rspec level using tags here https://github.com/elastic/logstash-devutils/blob/15c59bfaab592f22531d333a25e4dcac39256f18/lib/logstash/devutils/rspec/spec_helper.rb#L56-L60

when this file is loaded, only rspec knows it will exclude it, so it does not make sense to log anything at the top level because "we don't know" yet.